tests/DotNetLightning.Core.Tests/Generators/Msgs.fs

```diff
@@ -152,7 +155,10 @@ let acceptChannelGen =
         <*> delayedPaymentBasepointGen
         <*> htlcBasepointGen
         <*> perCommitmentPointGen
-        <*> (Gen.optionOf pushScriptGen)
+        <*> gen {
+                let! shutdownScript = (Gen.optionOf pushScriptGen)
+                return [| AcceptChannelTLV.UpfrontShutdownScript shutdownScript |]
```

Comment:
TLV fields generator should include GenericTLV field. 
So this generator should be something like
```fs
Gen.oneOf [
    genericTLVGen([ 0UL ])
    gen {
      let! shutdownScript = (Gen.optionOf pushScriptGen)
      return [| AcceptChannelTLV.UpfrontShutdownScript shutdownScript |]
  }
]
```

Reply:
@joemphilips updated